ndk/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 - native_window: Require linking against `libnativewindow` for most API >= 26 functions. (#465)
 - **Breaking:** audio: Merge `AudioResult` variant enum into `AudioError`. (#467)
 - data_space: Add missing `DataSpaceRange::Unspecified` variant. (#468)
+- **Breaking:** looper: Require `Send` marker when adding fd event callbacks on `ForeignLooper`. (#469)
```

Comment:
As soon as I click merge, I realized I should have probably mentioned that `ThreadLooper::add_fd_with_callback()` was added without the `Send` bound, if already on the same thread.